mac.sh

```diff
@@ -161,40 +152,6 @@ install_bundler_1_17_3() {
   bundle config --global jobs $((number_of_cores - 1))
 }
 
-install_vim_config() {
```

Comment:
wow matt, big of you to remove `vim`

Reply:
I know that anyone that wants vim has been hand crafting the perfect config for a long time :)

Reply:
Matt doesn't even use a config, he's pure vanilla vim